dali/dfuXRefLib/dfuxreflib.cpp

```diff
@@ -958,16 +964,26 @@ static bool parseFileName(const char *name,StringBuffer &mname,unsigned &num,uns
         if (startsWith(name, prefix) && name[prefixLen] == PATHSEPCHAR)
         {
             mname.ensureCapacity(strlen(name));
-            mname.append(prefix).append(PATHSEPCHAR);
-            name += prefixLen + 1;
+            mname.append(prefix);
+            name += prefixLen;
+            stripeNum = 0;
             if (plane.getPropInt("@numDevices") > 1)
             {
-                if (*name&&*name!='d')
+                name++;
```

Comment:
this is skipping the PATHSEPCHAR that the code established was present on line 964.
Logically this name + 1 should be before the "if (plane.getPropInt("@numDevices") > 1)" block, which is how it was afaics, prior to most recent changes.
i.e. I think line 972 should be deleted and line 968 should be: 
```
name += prefixLen + 1;
```